.travis.yml

```diff
@@ -1,83 +1,25 @@
-      - python-simplejson
-      - python-serial
-      - python-yaml
-#     Search your packages here:
-#       https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
-#     For wkhtmltopdf, see the env section below
-
-# Sometimes complicated website repos need Compass & SaSS:
-#before_install:
-#  - rvm install ruby --latest
-#  - gem install bootstrap-sass
-#  - gem install compass --pre
 
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  # Set this variable to some version existing as linux-generic build on
-  # https://github.com/wkhtmltopdf/wkhtmltopdf/releases
-  # if you need to install wkhtmltopdf
-  # - WKHTMLTOPDF_VERSION="0.12.4"
-  # Set the above to install a `wkhtmltopdf` version that is not the one provided
-  # by the `pov-wkhtmltopdf` repo.
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
   - PHANTOMJS_VERSION="latest"
-  # The above line controls the PhantomJS version that is used for JS testing.
-  #   It is not necessary to include this value unless you are altering the default.
-  #   Use `OS` to skip the PhantomJS upgrade & use the system version instead.
-  - WEBSITE_REPO="1"
-  # Use the above line to install dependencies that are required for website repos:
-  # * SASS & Bootstrap-SASS
-  # * Compass
-  - TRANSIFEX_USER='transbot@odoo-community.org'
-  # This line contains the encrypted transifex password
-  # To encrypt transifex password, install travis ruby utils with:
-  #   $ gem install travis --user-install
-  # and use:
-  #   $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
-  # Secure list for current OCA projects is in https://github.com/OCA/maintainer-quality-tools/issues/194
-  - secure: PjP88tPSwimBv4tsgn3UcQAD1heK/wcuSaSfhi2xUt/jSrOaTmWzjaW2gH/eKM1ilxPXwlPGyAIShJ2JJdBiA97hQufOeiqxdkWDctnNVVEDx2Tk0BiG3PPYyhXPgUZ+FNOnjZFF3pNWvzXTQaB0Nvz8plqp93Ov/DEyhrCxHDs=
-  # Use the following lines if you need to manually change the transifex project slug or/and the transifex organization.
-  # The default project slug is owner-repo_name-version (with dash in the version string).
-  # The default organization is the owner of the repo.
-  # The default fill up resources (TM) is True.
-  # The default team is 23907. https://www.transifex.com/organization/oca/team/23907/
-  # - TRANSIFEX_PROJECT_SLUG=
-  # - TRANSIFEX_ORGANIZATION=
-  # - TRANSIFEX_FILL_UP_RESOURCES=
-  # - TRANSIFEX_TEAM=
 
   matrix:
   - LINT_CHECK="1"
-  - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
   - TESTS="1" ODOO_REPO="OCA/OCB"
-# either use the two lines above or the two below. Don't change the default if
-# it's not necessary (it is only necessary if modules in your repository can't
-# be installed in the same database. And you get a huge speed penalty in your
-# tests)
-#  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
-#  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
-
-virtualenv:
```

Comment:
Isn't this needed?

Reply:
It's not in the template anymore, so I guess not.